src/main/java/ninja/amp/ampmenus/menus/ItemMenu.java

```diff
@@ -127,7 +128,20 @@ public void setParent(ItemMenu parent) {
      * @param menuItem The {@link ninja.amp.ampmenus.items.MenuItem}.
      * @return The {@link ninja.amp.ampmenus.menus.ItemMenu}.
      */
+    @Deprecated
     public ItemMenu setItem(int position, MenuItem menuItem) {
```

Comment:
This method is redundant, because MenuItem inherits IMenuItem so it can be passed to the other method. This could be safely removed.